scripts/coccinelle/fix_use_of_NUM_ELEMENTS.cocci

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2019 Network RADIUS SAS.  Licenced under CC-by-NC 4.0.
+// Development of this scripts was sponsored by Network RADIUS SAS.
+// Author: Jorge Pereira (jpereira@freeradius.org)
+// Confidence: High
+// Comments: Fix use of NUM_ELEMENTS instead of random sizeof(T)/sizeof(E)
+// Options: --no-includes
```

Comment:
Can the comments give at least one example of the types of transformation that will be applied?  Some inline descriptions of the syntax would be useful too.